### PR TITLE
Release guide update for VSCode plugin

### PR DESCRIPTION
**Pull request by prvshah51**
Adding information about updating and releasing new version of ide-vscode to provide new version of Dafny to VSCode plugin users.

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
 9. If everything works you can, at your leisure do
 
         git branch -d <the branch name>
+
+## Updating and releasing a new version of VSCode plugin
+
+1. Build the Ide-vscode locally from https://github.com/dafny-lang/ide-vscode .
```
Comment (atomb) [inline RELEASE.md:149]:
```suggestion
1. Build ide-vscode locally from https://github.com/dafny-lang/ide-vscode .
```

File: RELEASE.md [hunk 2]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+
+2. Prepare the repository:
+
+       git checkout master
+       git pull origin
+       git checkout -b <some new branch name>
+
+3. add the new Dafny version number under `dafny.preferredVersion` list in file `package.json`  .
```
Comment (atomb) [inline RELEASE.md:157]:
```suggestion
3. Add the new Dafny version number to the `dafny.preferredVersion` list in the `package.json` file.
```

File: RELEASE.md [hunk 3]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+
+4. Update the value of constant `LatestVersion` In the file `src/constants.ts` to new version number (.
```
Comment (atomb) [inline RELEASE.md:159]:
```suggestion
4. Update the value of the constant `LatestVersion` in the file `src/constants.ts` to the new version number.
```

File: RELEASE.md [hunk 4]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+
+5. Before releasing a new version of the VSCode plugin, Make sure to add the Release notes  in file `CHANGELOG.md`
```
Comment (atomb) [inline RELEASE.md:161]:
```suggestion
5. Before releasing a new version of the VSCode plugin, make sure to add to the release notes in `CHANGELOG.md`
```

File: RELEASE.md [hunk 5]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+
+6. Select a new version number $VER for Plugin(e.g. "2.4.0") and change the value of the constant `version` in the file `package,json`
```
Comment (atomb) [inline RELEASE.md:163]:
```suggestion
6. Select a new version number `$VER` for the plugin (e.g. "2.4.0") and change the value of the constant `version` in the file `package.json`
```

File: RELEASE.md [hunk 6]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+   to $VER and commit the changes.
```
Comment (atomb) [inline RELEASE.md:164]:
```suggestion
   to `$VER` and commit the changes.
```

File: RELEASE.md [hunk 7]
```diff
@@ -143,3 +143,31 @@ with git commands and concepts is helpful.
...
+
+7. Then tag and push the changes:
+
+       git tag v<$VER>
+       git push v<$VER>
+
+8. Create a pull request and once merged ,A GitHub action will automatically run in reaction to the tag being pushed to master, which will create a new release in the VSCode marketplace.
```
Comment (atomb) [inline RELEASE.md:171]:
```suggestion
8. Create a pull request and, once merged, a GitHub action will automatically run in reaction to the tag being pushed to master, which will create a new release in the VSCode marketplace.
```

**Review by atomb — CHANGES_REQUESTED** [review]
The content looks good. I just suggested a few grammatical improvements.

**Review by atomb — APPROVED** [review]
Looks good!

**Comment by MikaelMayer** [comment]
Fantastic ! Thanks for taking care of documenting this process.

**Pull request merged**